classes/order/OrderHistory.php

```diff
@@ -571,7 +571,7 @@ public function sendEmail($order, $template_vars = false)
                     null,
                     $file_attachement,
                     null,
-                    _PS_MAIL_DIR_,
+                    $result['module_name'] ? _PS_MODULE_DIR_ . $result['module_name'] . '/mails/' : _PS_MAIL_DIR_,
```

Comment:
If you do this, you _force_ the email to be in the modules directory. But some modules use the Core templates. This is why the UI tests are failing.

The idea is good, but a module sending an order state email must be able to choose whether the mail folder should _PS_MAIL_DIR_ or its own mail module. Like a fallback.